react/scss/styles.scss

```diff
@@ -0,0 +1,114 @@
+.bookSearch {
+  max-width: 792px;
+  margin: 0 auto;
+}
+
+.bookItem {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 120px;
+  height: 232px;
+  margin: 35px;
+  padding: 3px;
+  cursor: pointer;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
```

Comment:
With Create React App, these prefixes aren't needed:
https://github.com/facebookincubator/create-react-app#why-use-this